tests/ui/generics/issue-95208-ignore-qself.stderr

```diff
@@ -2,9 +2,12 @@ error: expected `:` followed by trait or lifetime
   --> $DIR/issue-95208-ignore-qself.rs:6:88
    |
 LL | impl<T: Iterator> Struct<T> where <T as std:: iter::Iterator>::Item:: std::fmt::Display {
-   |                                                                    ---                 ^
-   |                                                                    |
-   |                                                                    help: use single colon: `:`
+   |                                                                                        ^
+   |
+help: use single colon
+   |
+LL | impl<T: Iterator> Struct<T> where <T as std:: iter::Iterator>::Item: std::fmt::Display {
```

Comment:
this could be a removal suggestion instead of a replacement suggestion

Reply:
This one is tricky, because this could have been written as `<T as std:: iter::Iterator>::Item  ::    std::fmt::Display`, but we only have the spans for `<T as std:: iter::Iterator>::Item` and `std::fmt::Display`.